src/devrules/cli_commands/branch.py

```
@@ -66,15 +138,13 @@ def _handle_forbidden_cross_repo_card(gh_project_item: Any, config: Any, repo_me
         if not actual:
             actual = "<unknown>"
 
-        typer.secho(
+        prompter.error(
             msg.CROSS_REPO_CARD_FORBIDDEN.format(actual, expected),
-            fg=typer.colors.RED,
         )
+        raise prompter.exit(code=1)
```

Ruff (0.14.14)

Abstract `raise` to an inner function

(TRY301)